server/pulp/server/webservices/views/serializers/__init__.py

```diff
@@ -149,6 +154,86 @@ def to_representation(self, instance):
         return representation
 
 
+class ModelSerializer(BaseSerializer):
+    """
+    Base serializer for Mongoengine Documents.
+    """
+
+    def to_representation(self, instance):
+        """
+        Transforms a Mongoengine Document into a serialized dictionary.
+        """
+        document_dict = {}
```

Comment:
Should this translate the iso8601 stringfield from datetime to the string representation?